packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "04085c80c2c84ea6b8375042d61797f7f46138fd",
-    "ref_origin" : "dcos-mesos-master-f179400b"
+    "ref": "9d1cb5ecba1a7921b067ad94092c3663adf19b24",
+    "ref_origin" : "dcos-mesos-1.2.0-rc1"
```

Comment:
Yes, the `ref` (SHA) must be the HEAD of the dcos-mesos branch (in `mesosphere/mesos`) that includes the necessary parts of Apache Mesos, plus the dcos-specific cherry-picks on top.
The `ref_origin` (branch) should be the branch in `mesosphere/mesos` that the `ref` lives in.